tests/TestCase/I18n/TimeTest.php

```diff
@@ -47,6 +50,10 @@ public function tearDown()
         Time::setTestNow($this->now);
         Time::$defaultLocale = $this->locale;
         Time::resetToStringFormat();
+
+        FrozenTime::setTestNow($this->frozenNow);
+        FrozenTime::$defaultLocale = $this->locale;
+        FrozenTime::resetToStringFormat();
```

Comment:
I think we will need a global function or another class that is cabal me of setting some of those static methods for all of the versions of our time classes :S